TRAMS-API/Models/Request/PostProjectsRequestModel.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace API.Models.Request
+{
+    public class PostProjectsRequestModel
+    {
+        /// <summary>
+        /// The Project Initiator's Full Name. Mandatory.
+        /// </summary>
+        /// <example>Joe Bloggs</example>
+        [Required]
+        public string ProjectInitiatorFullName { get; set; }
+
+        /// <summary>
+        /// The unique id of the project initiator. Mandatory.
+        /// </summary>
+        /// <example>joe.blogs@email.com</example>
+        [Required]
```

Comment:
Suggestion: If this is always likely to be an email address add a regex data annotation for it.

Reply:
It's not no, it might be a guid or the username